ui/src/app/pages/admin/workspace/admin-workspace.tsx

```diff
@@ -129,7 +131,13 @@ const AdminWorkspaceImpl = (props: Props) => {
           />
           <Accordion>
             <AccordionTab header='Research Purpose'>
-              <ResearchPurposeSection {...{ researchPurpose }} />
+              <ResearchPurposeSection
+                {...{ researchPurpose }}
+                showAIAN={showAIANReasearchPurpose(
+                  findCdrVersion(workspace?.cdrVersionId, cdrVersionStore.get())
```

Comment:
since we're protecting workspace with a `?`, we should also do that to the result of `findCdrVersion()`